agave_pyclient/agave_pyclient/agave.py

```diff
@@ -937,6 +937,18 @@ def set_flip_axis(self, x: int, y: int, z: int):
         # 46
         self.cb.add_command("SET_FLIP_AXIS", x, y, z)
 
+    def set_interpolation(self, x: int):
+        """
+        Set the volume sampling interpolation mode
+
+        Parameters
+        ----------
+        x: int
+            0 for nearest, 1 for linear
```

Comment:
just curious: is it just always easier to deal with ints? if I was writing this in typescript I would use the names "nearest" and "linear" so that it was easy to know what the settings were throughout the code. Or have it be a bool, since this feature is just enabling or disabling a method

Reply:
Yes from an API design standpoint this is not good. 

The reason it's like this is because these commands that come from the python client or the js web client are getting converted to binary data stream to be sent to the c++ code via network (websocket).  And the client/server system I set up can deal with basic types like ints, floats, and arrays of ints and floats.  (Strings are also done as array of bytes)

So when you see these ints-as-booleans creep into the apis, it's just because it keeps things consistent across python, c++, and js codebases with the identical function signature.
A nice-to-have code change would be to set up good symbolic enums for the functions that take data types like this one, or booleans, and convert internally to the integer types. Then the external python and js apis could be more user friendly.